gulp/tasks/default.js

```diff
@@ -4,7 +4,7 @@ var runSequence     = require('run-sequence');
 gulp.task( 'default', function() {
     runSequence(
         ['styles', 'images'],
-        ['pl_styles', 'pl_scripts', 'pl_images'],
+        ['pl_styles', 'pl_scripts', 'scripts', 'pl_images'],
```

Comment:
NIT: move the ``scripts`` task to be within the rest of the general pattern tasks, like so:

```js
...
runSequence(
    ['styles', 'scripts', 'images'],
    ['pl_styles', 'pl_scripts', 'pl_images'],
...
```